src/Blocks/custom/forms/components/forms-editor.js

```diff
@@ -94,34 +84,23 @@ export const FormsEditor = ({
 
 			{isGeoPreview &&
 				<>
+					<div className='es-mt-5 es-text-7 es-text-align-center es-font-weight-700'>
+						{__('Geolocation alternatives', 'eightshift-forms')}
+					</div>
 					{formsFormGeolocationAlternatives.map((item, index) => {
 						return (
-							<>
-								<div className='es-mt-20 es-text-7 es-text-align-center es-font-weight-700 es-mb-3'>
-									{__('Geolocation alternative', 'eightshift-forms')}
-								</div>
-								<div className='es-mb-8 es-text-4 es-text-align-center'>
-									{item.geoLocation.join(', ')}
-								</div>
-								<ServerSideRender
-									key={index}
-									block={blockFullName}
-									attributes={
-										getFilteredAttributes(
-											attributes,
-											[
-												...attributesSsr,
-												'formsFormGeolocation',
-												'formsFormGeolocationAlternatives',
-											],
-											{
-												formsFormPostId: item.formId,
-												formsServerSideRender: true
-											}
-										)
-									}
-								/>
-							</>
+							<Placeholder
+								key={index}
+								icon={icons.form}
+								label={<span className='es-font-weight-400'>{__('Eightshift Forms', 'eightshift-forms')}</span>}
+								className='es-rounded-3! es-mt-5! es-mx-auto! es-font-weight-400 es-color-cool-gray-500! es-nested-color-current!'
```

Comment:
These should be replaced with TW at some point, but let's leave them as-is for now 😄